app/posts/claim-general-mentor-training-funding/2022-06-26-general-mentor-training-grant-guidance-and-claim-form.md

```diff
@@ -0,0 +1,120 @@
+---
+title: General mentor training grant guidance and claim form
+description: Guidance page summarising ITT gerneral mentor funding and claim form.
+date: 2022-06-26
+---
+
+## ITT market reform
```

Comment:
It's a bit unclear going straight from the page title h1 to this h2 heading. Drop the heading, or add an intro para?

Reply:
i think its fine to have heading followed by a head, but I've removed it